index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Portfolio</title>
+</head>
+<body>
+    <header>
+        <div id="welcome">
+            Welcome
```

Comment:
### Your Design
![white-spacing (copy)](https://user-images.githubusercontent.com/93039359/186410117-4f63ddc8-4dc8-4a3d-8826-bedeba5f9f26.png)

- [ ] Please remove this white spacing shown in the image above. I have given quick fix below :arrow_down:
### Quick Fix
#### 1.
![responsiveness](https://user-images.githubusercontent.com/93039359/186410479-d03dacb1-ef4e-48a5-9623-9b1187ea65f0.png)